src/sass/facility-locator.scss

```diff
@@ -71,6 +71,12 @@
       color: $color-link-default;
       text-decoration: none;
 
+      &:focus {
+        h5 {
+          border: $color-link-default solid thin;
```

Comment:
It's usually the `outline` property that's used for focus, right? Does that not work for IE?